### PR TITLE
Add 2.1.8 known issues

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

**Comment by dougbu** [comment]
@vivmishra should this new file be linked to from anything else?

File: release-notes/2.1/2.1.8/2.1.8-known-issues.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+```
+Unhandled Exception: System.IO.FileLoadException: Could not load file or assembly 'Microsoft.AspNetCore, Version=2.1.7.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
+   at Program.CreateWebHostBuilder(String[] args)
+```
+
+#### Workarounds
+
+If this issue affects your application, update the project in one of the following ways:
+1. Remove version from AspNetCore.App `<PackageReference Include="Microsoft.AspNetCore" />`
+2. add `<PackageReference Include="Microsoft.AspNetCore" Version="2.1.7" />` (NB: this is **not** the .App package)
```
Comment (vivmishra) [inline release-notes/2.1/2.1.8/2.1.8-known-issues.md]: Capitalize "add".

File: release-notes/2.1/2.1.8/2.1.8-known-issues.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+common.
+```
+Unhandled Exception: System.IO.FileLoadException: Could not load file or assembly 'Microsoft.AspNetCore, Version=2.1.7.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
+   at Program.CreateWebHostBuilder(String[] args)
+```
+
+#### Workarounds
+
+If this issue affects your application, update the project in one of the following ways:
+1. Remove version from AspNetCore.App `<PackageReference Include="Microsoft.AspNetCore" />`
```
Comment (omajid) [inline release-notes/2.1/2.1.8/2.1.8-known-issues.md]: The note says **.App** but the Include package doesn't include that. Am I just reading it wrong or is it a typo?

Comment (dougbu) [reply release-notes/2.1/2.1.8/2.1.8-known-issues.md]: @natemcmaster this was a typo in the list of workarounds I copied. Correct?

Comment (natemcmaster) [reply release-notes/2.1/2.1.8/2.1.8-known-issues.md]: oops, yes. 
```suggestion
1. Remove version from AspNetCore.App `<PackageReference Include="Microsoft.AspNetCore.App" />`
```

**Pull request merged**